homepage/templates/faq.html

```diff
@@ -0,0 +1,67 @@
+
+                    <h2>I have a question concerning data mining, statistics and machine learning algorithms.</h2>
+                    <p class="indent">Visit <a href="https://datascience.stackexchange.com/questions/tagged/orange">Data Science StackExchange</a> and ask the community!</p>
+
+                    <h2>What data does Orange support?</h2>
+                    <p class="indent">Core Orange supports Excel, comma- and tab-delimited files (.xlsx, .csv, .tab). It also reads online data, such as Google Spreadsheets. Orange3-ImageAnalytics add-on can import images (.jpg, .tiff, .png). Orange3-Corpus add-on can import text files (.txt, .docx, .odt). SQL widget supports PostgreSQL and MSSQL databases.</p>
+
+                    <h2>I want to use Orange on big data.</h2>
+                    <p class="indent">Orange provides the SQL widget, which samples data instances and enables exploratory data analysis on large datasets. You need <a href="http://initd.org/psycopg/">psycopg2</a> module installed to be able to see the widget. Install it directly in Python with <code>pip install psycopg2</code> command. You can also use <a href="https://www.2ndquadrant.com/en/resources/2uda/">2UDA package</a> for easier installation. Please note that Orange supports only PostgreSQL and MSSQL databases for now.</p>
+
+                    <h2>Can I export Orange workflow as a Python script?</h2>
+                    <p class="indent">Unfortunately, no. We've <a href="https://github.com/biolab/orange3/issues/1341">debated this long and hard</a>. A (limited) functionality of this type would probably be possible, but would be a very big project to do well and would cost more than the expected benefits.</p>
+
+                    <h2>I wish to propose a collaboration and/or a project with the Orange team.</h2>
+                    <p class="indent">Please submit a contact request and we will respond as soon as we can.</p>
+
+                    <h2>I wish to develop a widget / an add-on for Orange.</h2>
+                    <p class="indent">We are always happy to receive contributions! Development documentation for <a href="http://orange-development.readthedocs.io/">widgets</a> and <a href="https://github.com/biolab/orange3-example-addon">add-ons</a> is available on the web. Next, submit a pull request and our team will review it. If you have specific questions, use <a href="https://gitter.im/biolab/orange3">Gitter</a> for direct communication with the developers.</p>
+
+                    <h2>I wish to commission a custom module and/or an add-on for Orange.</h2>
+                    <p class="indent">Please submit a contact request to discuss the idea and the rates for commission.</p>
```

Comment:
> Discuss the idea.

Trim zeal for commission.

Reply:
No idea what you mean.

Reply:
Edited. Imho not nice to so openly show interest in acquiring money. It's already implied in the question, and good faith is assumed among gentlemen.

Reply:
Another commenter was concerned the original sentence was not explicit enough, so I fixed it. This also goes in line with all the questions coming to our mail, that rarely understand certain features aren't free. I see no shame in recognizing widgets don't code themselves out of thin air.